portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,23 +28,30 @@
 /** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  ArrayList<String> myComments = new ArrayList<>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    List<String> comments = new ArrayList<>();
-    comments.add("Comment 1: This blog is interesting!");
-    comments.add("Comment 2: Well known information. Not too interesting.");
-    comments.add("Comment 3: Needs some more development, but cool concept!");
-    String commentJSON = convertToJson(comments);
+    String commentJSON = convertToJson(myComments);
 
     //Send JSON as the response
     response.setContentType("application/json;");
     response.getWriter().println(commentJSON);
   }
 
-  private static String convertToJson(List<String> comments) {
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    //Get response from the form
```

Comment:
```suggestion
    // Get response from the form
```

to match [comment style](https://google.github.io/styleguide/javaguide.html#s4.8.6-comments)